ansible/roles/proteus-notify/defaults/main.yml

```diff
@@ -0,0 +1,37 @@
+go_arch: amd64
+
+proteus_user:   proteus
+proteus_group:  proteus
+
+proteus_install_path: /srv/proteus
+proteus_data_path: /srv/proteus/data
+proteus_config_path: /srv/proteus/config
+
+proteus_environment: "production"
+proteus_log_level: "info"
+proteus_api_port: 9081
+proteus_api_address: "127.0.0.1"
+proteus_notify_https_port: 8081
+proteus_database_url: "postgres://proteus:{{ proteus_database_password }}@{{ proteus_db_hostname }}:5432/proteus?sslmode=require"
+proteus_auth_jwt_secret : "{{ proteus_auth_jwt_secret }}"
+
+proteus_fcm_server_key: "{{ fcm_server_key }}"
+proteus_apn_key_path: "{{ proteus_config_path }}/apn-key.pem"
+proteus_apn_key_password: "{{ proteus_apn_key_password }}"
+proteus_apn_production: "{{ proteus_environment == 'production' }}"
+proteus_apn_key_content: "{{ proteus_apn_key_content }}"
+
+proteus_version: "0.1.0-beta.9"
+proteus_download_url: https://github.com/thetorproject/proteus/releases/download
+
+proteus_tarball_url: "{{ proteus_download_url }}/v{{ proteus_version }}/proteus-notify-{{ proteus_version }}.linux-{{ go_arch }}.tar.gz"
+proteus_path: "{{ proteus_install_path }}/proteus-notify-{{ proteus_version }}.linux-{{ go_arch }}"
+
+gorush_version: "1.9.0"
+gorush_download_url: "https://github.com/appleboy/gorush/releases/download"
+gorush_url: "{{ gorush_download_url }}/v{{ gorush_version }}/gorush-v{{ gorush_version }}-linux-{{ go_arch }}"
+gorush_path: "{{ proteus_install_path }}/gorush-v{{ gorush_version }}-linux-{{ go_arch }}"
+
+notification_backend: "gorush"
```

Comment:
Should code for `notification_backend: proteus` be removed and corresponding binaries and services cleaned up?